lib/commands/test.js

```diff
@@ -27,6 +27,7 @@ module.exports = Command.extend({
     { name: 'launch',      type: String,  default: false,                            description: 'A comma separated list of browsers to launch for tests.' },
     { name: 'reporter',    type: String,                            aliases: ['r'],  description: 'Test reporter to use [tap|dot|xunit] (default: tap)' },
     { name: 'silent',      type: Boolean, default: false,                            description: 'Suppress any output except for the test report' },
+    { name: 'debug',       type: String,                                             description: 'File to write a debug log from testem' },
```

Comment:
should this maybe be `--testem-debug` ? `--debug` is fairly vague, and would love to reserve this for some future general ember-cli `--debug` mode.